lib/gollum/app.rb

```diff
@@ -44,6 +45,8 @@ class App < Sinatra::Base
     register Mustache::Sinatra
     register Sinatra::Namespace
     include Precious::Helpers
+    include Precious::Views::AppHelpers
```

Comment:
needed the `extract_page_dir(file)` helper below. It's a bit ugly to include view helpers here. Ideally, the feed generation would be part of a view. We could do so by registering our own Tilt templates. On the other hand, this works, and I don't know how big a deal it is.

Reply:
Why would the view need to be a Tilt template?

Reply:
The route would then just look like this:

```ruby
get '/feed'
@changes = wiki_new.latest_changes(...)
rss :index
```

And all the logic for generating the actual content would be moved to the view.